connection/engine.go

```diff
@@ -125,7 +115,7 @@ func (e *Engine) InitializePeer(wgPort int, myKey wgtypes.Key, peer Peer) {
 func (e *Engine) ClosePeerConnection(peer Peer) error {
```

Comment:
I think we should rename it to something like RemovePeer because we are not only closing the connection but also removing a peer completely
```suggestion
func (e *Engine) RemovePeer(peer Peer) error {
```

Reply:
Renamed to RemovePeerConnection, removing Peer from a config file or management client will be the job of another process